platform/nativescript/compiler/modules/view.js

```diff
@@ -11,8 +11,8 @@ function transformNode(el) {
       mods[mod] = true
       return mods
     }, {})
-    getAndRemoveAttr(el, attr)
-    addDirective(el, 'view', `v-view:${attrName}`, '', arg, modifiers)
+    getAndRemoveAttr(el, attr, true)
+    addDirective(el, 'view', `v-view:${attrName}`, '', false, arg, modifiers)
```

Comment:
@farfromrefug what do the added args do?  

The first one is `removeFromMap` - why is this required?

The second one seems to be mapping to `arg` - is this the intended parameter? Or should that be `isDynamicArg` instead?

```js
export function addDirective (
  el: ASTElement,
  name: string,
  rawName: string,
  value: string,
  arg: ?string,
  isDynamicArg: boolean,
  modifiers: ?ASTModifiers,
  range?: Range
)
```

Reply:
oh my bad i inversed the args. Maybe that's why arg was not working anymore! will fix

Reply:
@farfromrefug perfect, hopefully that will fix it!

Reply:
done